homeassistant/components/sensor/wunderground.py

```diff
@@ -157,7 +185,15 @@ def update(self):
                                  ["description"])
             else:
                 self.data = result["current_observation"]
+
+            result = requests.get(self._build_url(_ALERTS), timeout=10).json()
+            if "error" in result['response']:
+                raise ValueError(result['response']["error"]
```

Comment:
This means that both `self.data` and `self.alerts` get set to `None` although only alerts failed